src/components/inputtools/TextStyle.vue

```diff
@@ -1,40 +1,35 @@
-                 data-code="00" @click="onColourClick"/>
-            <div class="kiwi-inputtools-colours-colour irc-bg-colour-black"
-                 data-code="01" @click="onColourClick"/>
-            <div class="kiwi-inputtools-colours-colour irc-bg-colour-blue"
-                 data-code="02" @click="onColourClick"/>
-            <div class="kiwi-inputtools-colours-colour irc-bg-colour-green"
-                 data-code="03" @click="onColourClick"/>
-            <div class="kiwi-inputtools-colours-colour irc-bg-colour-light-red"
-                 data-code="04" @click="onColourClick"/>
-            <div class="kiwi-inputtools-colours-colour irc-bg-colour-brown"
-                 data-code="05" @click="onColourClick"/>
-            <div class="kiwi-inputtools-colours-colour irc-bg-colour-purple"
-                 data-code="06" @click="onColourClick"/>
-            <div class="kiwi-inputtools-colours-colour irc-bg-colour-orange"
-                 data-code="07" @click="onColourClick"/>
-            <div class="kiwi-inputtools-colours-colour irc-bg-colour-yellow"
-                 data-code="08" @click="onColourClick"/>
-            <div class="kiwi-inputtools-colours-colour irc-bg-colour-light-green"
-                 data-code="09" @click="onColourClick"/>
-            <div class="kiwi-inputtools-colours-colour irc-bg-colour-cyan"
-                 data-code="10" @click="onColourClick"/>
-            <div class="kiwi-inputtools-colours-colour irc-bg-colour-light-cyan"
-                 data-code="11" @click="onColourClick"/>
-            <div class="kiwi-inputtools-colours-colour irc-bg-colour-light-blue"
-                 data-code="12" @click="onColourClick"/>
-            <div class="kiwi-inputtools-colours-colour irc-bg-colour-pink"
-                 data-code="13" @click="onColourClick"/>
-            <div class="kiwi-inputtools-colours-colour irc-bg-colour-grey"
-                 data-code="14" @click="onColourClick"/>
-            <div class="kiwi-inputtools-colours-colour irc-bg-colour-light-grey"
-                 data-code="15" @click="onColourClick"/>
-            <div class="kiwi-inputtools-colours-colour kiwi-inputtools-colours-reset"
-                 @click="onResetClick"><i class="fa fa-ban" aria-hidden="true"/></div>
+            <div class="kiwi-inputtool-colour-actions" >
+                <div class="kiwi-inputtool-button" @click="showModernPalette = !showModernPalette">
+                    <i v-if="!showModernPalette" class="fa fa-chevron-down" aria-hidden="true" />
+                    <i v-else class="fa fa-chevron-up" aria-hidden="true" />
+                </div>
+            </div>
+
+            <div class="kiwi-inputtools-colour-container">
+                <div :class="{active: !showModernPalette}" class="kiwi-inputtools-colours">
+                    <div v-for="n in 15"
+                         :key="n"
+                         :class="['irc-bg-colour-' + n]"
+                         :data-code="[n]"
```

Comment:
Any reason this value is in an array now?

Reply:
It's simply looping over the N value - the 'array' brackets are what google said was best practice for concating a predefined string with a looping var :)  

Not sure if :class="{irc-bg-colour + 'n'}" was valid, so turned to google